flank-scripts/src/main/kotlin/flank/scripts/shell/ops/BuildTestPlansExample.kt

```diff
@@ -0,0 +1,32 @@
+package flank.scripts.shell.ops
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import flank.scripts.shell.utils.failIfWindows
+import flank.scripts.shell.utils.iOSTestProjectsPath
+import java.nio.file.Paths
+
+object BuildTestPlansExample : CliktCommand(name = "build_ios_testplans_example", help = "Build ios test plans example app") {
```

Comment:
please update documentation in README.md of flank-scripts

Reply:
Updated